hugegraph-llm/src/hugegraph_llm/demo/rag_web_demo.py

```diff
@@ -228,6 +246,37 @@ def apply_embedding_config(arg1, arg2, arg3, origin_call=None) -> int:
     return status_code
 
 
+def apply_reranker_config(arg1, arg2, arg3: str | None = None, origin_call=None) -> int:
```

Comment:
if `arg1~n` is not flexible, better to use a real name for it (like `api_key`...)